hearpreprocess/tfds_speech_commands.py

```diff
@@ -14,6 +14,7 @@
     "version": "0.0.2",
```

Comment:
Can we make a separate PR that is just all the simple mechanical changes like "trainvaltest" so we can merge that quickly? Not a huge deal, but if it's easy let's do it so we can zoom in on the stuff that is stuff confusing